Forms/Container.cs

```diff
@@ -204,7 +204,9 @@ private void refreshProcessList()
             });
             foreach (Process p in Process.GetProcesses())
             {
-                if (p.MainWindowTitle != "")
+                bool isGameProcess = ClientListSingleton.ExistsByName(p.ProcessName);
+
+                if (p.MainWindowTitle != "" && isGameProcess)
```

Comment:
Call directly `ClientListSingleton.ExistsByName(p.ProcessName)`

Reply:
@Demnokkoyen just a small suggestion, once you address this, we can merge the PR. Thanks for the contribution! 👍

Reply:
@afonsocdaniel fixed it! Just a small suggestion: DON'T merge this PR before setting every processName property in `supported_servers.json`. Servers without the property set to a valid value won't work be shown inside the process selectbox; I'm asking for your help cause I don't have all of these servers installed on my machine.

Reply:
You don't need this extra property. You can use execName that already exists in Client.